python_modules/dagster/dagster_tests/daemon_sensor_tests/test_sensor_run.py

```diff
@@ -2908,10 +2908,11 @@ def test_repository_namespacing(executor):
             assert len(ticks) == 2
 
 
-def test_settings():
+@pytest.mark.parametrize("daemon", ["backfills", "schedules", "sensors"])
```

Comment:
nit: since this is no longer just a sensor settings test, maybe we should move it to a more general location